py-polars/polars/expr/expr.py

```diff
@@ -2363,7 +2363,9 @@ def index_of(self, element: IntoExpr) -> Expr:
         return self._from_pyexpr(self._pyexpr.index_of(element))
 
     def search_sorted(
-        self, element: IntoExpr | np.ndarray[Any, Any], side: SearchSortedSide = "any"
+        self,
+        element: IntoExpr | np.ndarray[Any, Any] | dict[str, Any],
```

Comment:
`dict[str, Any]`?

Reply:
That's how you do a `Struct`.

Maybe... `polars._typing.PythonLiteral` should get `dict[str, Any]` added to it?

Reply:
Mmmh, weird. That should be in PythonLiteral.